### ISSUE TITLE
Add CODEOWNERS file

### ISSUE THREAD

**Issue by halkeye** [jenkins-infra/jenkins-wiki-exporter#12]
Do we have a team yet?

Probably should be at least me

**Comment by oleg-nenashev** [comment]
+1 for docs. CC @MarkEWaite

**Comment by MarkEWaite** [comment]
Yes, please, let's use https://github.com/jenkins-docs as the team.

**Comment by halkeye** [comment]
Can we use an org as a codeowner? I think it has to be a member of Jenkins infra

**Comment by oleg-nenashev** [comment]
Yes, we can only use a team with at least explicit Read permissions in the
repo

**Comment by halkeye** [comment]
Okay, I'm merging https://github.com/jenkins-infra/jenkins-wiki-exporter/pull/16/files with me and infra/docs team
